Source/Device.swift.gyb

```diff
@@ -363,6 +364,21 @@ public enum Device {
   public var localizedModel: String {
     return UIDevice.current.localizedModel
   }
+
+  /// PPI (Pixels per Inch) on the current device's screen (if applicable). When not applicable the property returns nil.
+  public var ppi: Int? {
+    #if os(iOS)
+    switch self {
+      % for device in iOSDevices:
+      case .${device.caseName}: return ${device.ppi}
+      % end
+      default:
```

Comment:
You would insert a `case .simulator(let device): return device.ppi` between the %end and the default, or I think the default would be unnessecary then and we should remove it so that we get a compiler error when a device is not handled in that switch statement.